File: src/main/java/plus/maa/backend/controller/response/SubCommentsInfo.java
package plus.maa.backend.controller.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import java.util.Date;

/**
 * @author LoMu
 * Date  2023-02-20 17:05
 */

@Data
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
public class SubCommentsInfo {
    private String commentId;
    private String uploader;
    private String uploaderId;

    //评论内容
    private String message;
    private Date uploadTime;
    private int like;
    private String fromCommentId;
    private String mainCommentId;
    private boolean deleted;
}

File: src/main/java/plus/maa/backend/controller/response/comments/SubCommentsInfo.kt
package plus.maa.backend.controller.response.comments

import java.time.LocalDateTime

/**
 * @author LoMu
 * Date  2023-02-20 17:05
 */
data class SubCommentsInfo(
    private val commentId: String,
    private val uploader: String,
    private val uploaderId: String,
    //评论内容,
    private val message: String,
    private val uploadTime: LocalDateTime,
    private val like: Long = 0,
    private val dislike: Long = 0,
    private val fromCommentId: String,
    private val mainCommentId: String,
    private val deleted: Boolean = false
)
